app/test/specs/inviteInstitutionControllerSpec.js

```diff
@@ -94,31 +94,30 @@
                 scope.$apply();
             });
 
-            it('should call InviteinstitutionCtrl.sendInvite() and InstitutionService.searchInstitutions()',
-                function(done) {
-                    spyOn(instService, 'searchInstitutions').and.callThrough();
-                    spyOn(inviteinstitutionCtrl, 'sendInstInvite');
-                    inviteinstitutionCtrl.invite.invitee = "mayzabeel@gmail.com";
-                    inviteinstitutionCtrl.invite.suggestion_institution_name = "New Institution";
-                    inviteinstitutionCtrl.user.current_institution = splab;
-                    httpBackend.expect('GET', "api/search/institution?name=New Institution&state=(active OR pending)").respond({});
-                    inviteinstitutionCtrl.checkInstInvite().then(function() {
-                        expect(instService.searchInstitutions).toHaveBeenCalledWith(
-                            inviteinstitutionCtrl.invite.suggestion_institution_name,
-                            "(active OR pending)");
-                        expect(inviteinstitutionCtrl.sendInstInvite).toHaveBeenCalled();
-                        done();
-                    });
-                    httpBackend.flush();
+            it('should call sendInvite() and searchInstitutions()', function(done) {
+                spyOn(instService, 'searchInstitutions').and.callThrough();
+                spyOn(inviteinstitutionCtrl, 'sendInstInvite');
+                inviteinstitutionCtrl.invite.invitee = "mayzabeel@gmail.com";
```

Comment:
ao invés de fazer toda a atribuição nas propriedades cria um objeto invitee dentro do it e atribui direto. ex:

```
var invite = {invitee: "mayza@...", suges...: "new institution"}
inviteCtrl.invite = invite
```